assets/src/blocks/events-list/edit.js

```diff
@@ -166,11 +170,84 @@ const Edit = ( props ) => {
 						maxSuggestions={ 20 }
 					/>
 				</PanelBody>
+				<PanelBody>
+					<ToggleControl
+						label="Show/Hide Attendee list"
+						help={
+							attributes.showAttendeeList
+								? 'Show Attendee List'
+								: 'Do not show Attendee List'
```

Comment:
Internationalize lines 178 and 179 (see example above)

Reply:
@mauteri done.